docs/sources/mimir/release-notes/v2.6.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Grafana Mimir version 2.6 release notes"
+menuTitle: "V2.6 release notes"
+description: "Release notes for Grafana Mimir version 2.6"
+weight: 300
+---
+
+# Grafana Mimir version 2.6 release notes
+
+Grafana Labs is excited to announce version 2.6 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Lower memory usage in store-gateway by streaming series results**
+  The store-gateway can now stream results back to the querier instead of buffering them. This is expected to greatly reduce peak memory consumption while keeping latency the same. This is still an **experimental feature** and is subject to a changing API and instability.
+
+- **Improved stability in store-gateway by removing mmap usage**
+  The store-gateway can now use an alternate code path to read index-headers that does not use memory mapped files. This is expected to improve stability of the store-gateway. This is still an **experimental feature** and is subject to a changing API and instability.
+
+- **Alertmanager Webex support**
+  Alertmanager can now use Webex to send alerts.
+
```

Comment:
Can you add an entry about all features moved to stable? The 2 you recently did (the CHANGELOG entry for them is still missing) and this one:

> * [ENHANCEMENT] Query-frontend: CLI flag `-query-frontend.max-total-query-length` and its associated YAML configuration is now stable. #3882